src/control/server/engine/utils.go

```diff
@@ -98,10 +107,53 @@ func ValidateLogMasks(masks string) error {
 			return errors.Errorf("illegal log mask assignment: want PREFIX=LEVEL got %q",
 				tok)
 		}
-		if !isValidLevel(facLevel[1]) {
+		if !isLogLevelValid(facLevel[1]) {
 			return errUnknownLogLevel(facLevel[1])
 		}
 	}
 
 	return nil
 }
+
+var (
+	validLogStreams = []string{
+		"ALL",                                                      // Select all debug streams
+		"MD", "PL", "MGMT", "EPC", "DF", "REBUILD", "DAOS_DEFAULT", // DAOS debug streams
+		"ANY", "TRACE", "MEM", "NET", "IO", // GURT debug streams
+	}
+	errAllWithOtherLogStream = errors.New("all stream identifier cannot be used with any other")
```

Comment:
minor - might want to put "all" in quotes to ensure the meaning is clear to users

Reply:
done